tls/s2n_kem_preferences.c

```diff
@@ -37,6 +37,22 @@ const struct s2n_kem *pq_kems_r2r1_2020_07[5] = {
     &s2n_sike_p503_r1,
 };
 
+const struct s2n_kem *pq_kems_r3r2r1_2021_05[8] = {
+    /* Round 3 Algorithms */
+    &s2n_kyber_512_r3,
+    &s2n_bike_l1_r3,
+    &s2n_sike_p434_r3,
+
+    /* Round 2 Algorithms */
+    &s2n_kyber_512_r2,
+    &s2n_bike1_l1_r2,
+    &s2n_sike_p434_r2,
```

Comment:
I'd suggest leaving sikep434r2 off of all new lists for both pq 1.2 and 1.3. sike_r2 and sike_r3 are compatible specs, so r2 is obsoleted by r3. We don't sacrifice anything by leaving off r2.